themes/dark/templates/editShow.mako

```diff
@@ -263,19 +308,41 @@ var startVue = function() {
                         paused: this.seriesObj.config.paused,
                         location: this.seriesObj.config.location,
                         airByDate: this.seriesObj.config.airByDate,
-                        subtitlesEnabled: this.seriesObj.config.subtitlesEnabled
+                        subtitlesEnabled: this.seriesObj.config.subtitlesEnabled,
+                        release: {
+                            requiredWords: this.seriesObj.config.release.requiredWords,
+                            ignoredWords: this.seriesObj.config.release.ignoredWords
+                            // blacklist: this.seriesObj.config.release.blacklist,
+                            // whitelist: this.seriesObj.config.release.whitelist
+                        }
+                            
                     }
                 }).then(response => {
                     log.info(response);
                 }).catch(err => {
                     log.error(err);
                 });
+            },
+            onChangeIgnoredWords: function(items) {
+		        console.log('Event from child component emitted', items);
+                this.seriesObj.config.release.ignoredWords = items.map(item => {return item.value});
```

Comment:
Change to `this.seriesObj.config.release.ignoredWords = items.map(item => item.value);`.

Reply:
Or even `this.seriesObj.config.release.ignoredWords = items.map(({value}) => value);``